### PR TITLE
add lock, panel and kie editor tests

### PR DESCRIPTION
**Pull request by barboras7**

### CODE REVIEW

**Comment by ederign** [comment]
jenkins retest this

**Comment by barboras7** [comment]
jenkins retest this

**Comment by ederign** [comment]
@barboras7 thanks for your PR. Could you please fix the code formating?

File: uberfire-api/src/test/java/org/uberfire/workbench/model/impl/PanelDefinitionImplTest.java [hunk 1]
```diff
@@ -16,29 +16,37 @@
...
         panelDefinition = new PanelDefinitionImpl();
         otherPanel = new PanelDefinitionImpl();
+        parent = new PanelDefinitionImpl();
         placeRequest = mock( PlaceRequest.class );
         part = new PartDefinitionImpl( placeRequest );
     }
 
+    @Test(expected = IllegalStateException.class)
+    public void settingTwoDifferentParentsShouldThrowException(){
+        panelDefinition.setParent(otherPanel);
```
Comment (ederign) [inline uberfire-api/src/test/java/org/uberfire/workbench/model/impl/PanelDefinitionImplTest.java]: panelDefinition.setParent( otherPanel );

**Comment by psiroky** [comment]
@ederign what formatting are you expecting? Just the spaces around the method params? I find it rather disturbing that we still don't really have an unified code formatting style :(

**Comment by ederign** [comment]
@psiroky at first yes. And yes, +1 for unified code formatting style

**Comment by barboras7** [comment]
jenkins retest this

**Comment by csadilek** [comment]
jenkins retest this

**Comment by csadilek** [comment]
Test failure unrelated to this PR (and not reproducible locally). 

JIRA: https://issues.jboss.org/browse/UF-450

jenkins retest this.

**Comment by csadilek** [comment]
Thanks @barboras7. 

Merged now: 03221d3a2ad8da0d8a8b629e157ef9ea18f6f747

I took care of some minor remaining formatting fixes.

**Pull request closed without merging**